pkgs/development/ocaml-modules/ppx_blob/default.nix

```diff
@@ -9,10 +9,11 @@ buildDunePackage rec {
     sha256 = "1xmslk1mwdzhy1bydgsjlcb7h544c39hvxa8lywp8w72gaggjl16";
   };
 
-  buildInputs = [ alcotest ocaml-migrate-parsetree ];
-  doCheck = true;
+  checkInputs = lib.optional doCheck alcotest;
+  buildInputs = [ ocaml-migrate-parsetree ];
+  doCheck = lib.versionAtLeast ocaml.version "4.03";
 
-  meta = with stdenv.lib; {
+  meta = with lib; {
     homepage = https://github.com/johnwhitington/ppx_blob;
```

Comment:
Per https://github.com/NixOS/rfcs/pull/45, unquoted URLs are deprecated, so we're slowly cleaning them up as we go.  (Same for the `src` URL above.)